GVRf/Framework/framework/src/main/jni/vulkan/vulkan_flags.cpp

```diff
@@ -0,0 +1,68 @@
+        glToVulkan[ GL_DST_COLOR ]                          = VkBlendFactor::VK_BLEND_FACTOR_DST_COLOR;
+        glToVulkan[ GL_ONE_MINUS_DST_COLOR ]                = VkBlendFactor::VK_BLEND_FACTOR_ONE_MINUS_DST_COLOR;
+        glToVulkan[ GL_SRC_ALPHA ]                          = VkBlendFactor::VK_BLEND_FACTOR_SRC_ALPHA;
+        glToVulkan[ GL_ONE_MINUS_SRC_ALPHA ]                = VkBlendFactor::VK_BLEND_FACTOR_ONE_MINUS_SRC_ALPHA;
+        glToVulkan[ GL_DST_ALPHA ]                          = VkBlendFactor::VK_BLEND_FACTOR_DST_ALPHA;
+        glToVulkan[ GL_ONE_MINUS_DST_ALPHA ]                = VkBlendFactor::VK_BLEND_FACTOR_ONE_MINUS_DST_ALPHA ;
+        glToVulkan[ GL_CONSTANT_COLOR ]                     = VkBlendFactor::VK_BLEND_FACTOR_CONSTANT_COLOR;
+        glToVulkan[ GL_ONE_MINUS_CONSTANT_COLOR ]           = VkBlendFactor::VK_BLEND_FACTOR_ONE_MINUS_CONSTANT_COLOR;
+        glToVulkan[ GL_CONSTANT_ALPHA ]                     = VkBlendFactor::VK_BLEND_FACTOR_CONSTANT_ALPHA;
+        glToVulkan[ GL_ONE_MINUS_CONSTANT_ALPHA ]           = VkBlendFactor::VK_BLEND_FACTOR_ONE_MINUS_CONSTANT_ALPHA;
+        glToVulkan[GL_SRC_ALPHA_SATURATE]                   = VkBlendFactor::VK_BLEND_FACTOR_SRC_ALPHA_SATURATE;
+
+
+        glToVulkan[GL_KEEP]                                 = VkStencilOp ::VK_STENCIL_OP_KEEP ;
+        glToVulkan[GL_ZERO]                                 = VkStencilOp::VK_STENCIL_OP_ZERO;
+        glToVulkan[GL_REPLACE]                              = VkStencilOp::VK_STENCIL_OP_REPLACE;
+        glToVulkan[GL_INCR]                                 = VkStencilOp::VK_STENCIL_OP_INCREMENT_AND_CLAMP;
+        glToVulkan[GL_INCR_WRAP]                            = VkStencilOp::VK_STENCIL_OP_INCREMENT_AND_WRAP ;
+        glToVulkan[GL_DECR]                                 = VkStencilOp::VK_STENCIL_OP_DECREMENT_AND_CLAMP;
+        glToVulkan[GL_DECR_WRAP]                            = VkStencilOp::VK_STENCIL_OP_DECREMENT_AND_WRAP;
+        glToVulkan[GL_INVERT]                               = VkStencilOp::VK_STENCIL_OP_INVERT;
+
+
+        glToVulkan[GL_NEVER]                                =VkCompareOp::VK_COMPARE_OP_NEVER;
+        glToVulkan[GL_LESS]                                 =VkCompareOp::VK_COMPARE_OP_LESS;
+        glToVulkan[GL_LEQUAL]                               =VkCompareOp::VK_COMPARE_OP_LESS_OR_EQUAL;
+        glToVulkan[GL_GREATER]                              =VkCompareOp::VK_COMPARE_OP_GREATER;
+        glToVulkan[GL_GEQUAL]                               =VkCompareOp::VK_COMPARE_OP_GREATER_OR_EQUAL;
+        glToVulkan[GL_EQUAL]                                =VkCompareOp::VK_COMPARE_OP_EQUAL;
+        glToVulkan[GL_NOTEQUAL]                             =VkCompareOp::VK_COMPARE_OP_NOT_EQUAL;
+        glToVulkan[GL_ALWAYS]                               =VkCompareOp::VK_COMPARE_OP_ALWAYS;
+
+    }
+
+    int VulkanConversionFlags::getFromGL(int GlFlag) {
+
+        return glToVulkan[GlFlag];
+    }
+}
```

Comment:
we can have static class for this or just a namespace.

Reply:
Also, please check if there are other conversion params from GL to Vulkan, lets add them here. we can use unordered_map.